src/libs/routing/A.tsx

```diff
@@ -0,0 +1,11 @@
+import { FC, HTMLProps } from 'react';
+
+type Props = HTMLProps<HTMLAnchorElement> & { to: string };
+
+export const A: FC<Props> = ({ to, children, ...props }) => {
```

Comment:
I think the name doesn't reflect the logic. Maybe something like `NewTabLink` or something like.